quartz/components/PageList.tsx

```diff
@@ -46,7 +46,7 @@ export function PageList({ cfg, fileData, allFiles, limit }: Props) {
             <div class="section">
               {page.dates && (
                 <p class="meta">
-                  <Date date={getDate(cfg, page)!} />
+                  <Date date={getDate(cfg, page)!} locale={cfg.locale ?? "en-US"} />
```

Comment:
```suggestion
                  <Date date={getDate(cfg, page)!} locale={cfg.locale} />
```

because we set the default param in `formatDate`, we can avoid defaulting here